### ISSUE TITLE
Stellarium crash on startup

### ISSUE THREAD

**Issue by Mikedafish** [Stellarium/stellarium#981]
### Expected Behaviour
'Normal operation'

### Actual Behaviour
On startup either by icon click or command line 'open ....' crash is reported before any kind of display.

### System
* Stellarium version: 0.19.3
* Operating system: MAC OS X High Sierra 10.14.6
* Graphics Card: NVIDIA GeForce GTX 780M
* 27 inch iMac end 2013 8Gbytes RAM
[stellarium_2020-02-27-231839_iMac27MichelP.crash.zip](https://github.com/Stellarium/stellarium/files/4264476/stellarium_2020-02-27-231839_iMac27MichelP.crash.zip)

Previous versions of Stellarium have run on this machine.
Installation is via MacPorts and following first crashes I updated all ports and reinstalled Stellarium.

### Logfile 
logfile attached

**Comment by alex-w** [comment]
I've tried install Stellarium though macports on 10.14.4 Mojave (High Sierra 10.14.6 - WHY???) and it running without problem.

**Comment by Mikedafish** [comment]
OK thank you for that.
It was obviously 10.14.6 Mojave that I am running. Sorry for misleading info!
I performed a thorough cleanup of my MacPorts (self update / upgrade outdated / uninstall stellarium / diagnose / reclaim (the last 2 were new to me) / install Stellarium)
and all is FINE!

Lesson learnt: keep a clean system. I recovered 479 Mb in the process and fixed my issue which was purely local.

Thanx again

M.
